tests/jballerina-unit-test/src/test/resources/test-src/statements/vardeclr/module_record_var_decl_taint_analysis_negetive.bal

```diff
@@ -0,0 +1,22 @@
+//  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+//  WSO2 Inc. licenses this file to you under the Apache License,
+//  Version 2.0 (the "License"); you may not use this file except
+//  in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// Test taint analysis
+record {int a;} {a:myA} = {a:5};
```

Comment:
Formatter doesn't add space after the colon in mapping bp. But It does adds space after colon in mapping constructor which is inconsistent. IMO we should remove the space in mapping constructor as well since it's better readability without space.

Reply:
The space between the colon and the value is specified in the Ballerina style guide - https://ballerina.io/swan-lake/learn/coding-conventions/expressions/#record-literal. Related discussion from when this was originally propsed - https://groups.google.com/g/ballerina-dev/c/3WqpIAuhS8w/m/7Z29oXugDQAJ.

Can you please create an issue for the difference with binding patterns?

Let's maybe initiate a separate discussion regarding changing it altogether.

Reply:
Created an issue #28472